dlt/common/runners/venv.py

```diff
@@ -22,6 +23,8 @@ def post_setup(self, context: types.SimpleNamespace) -> None:
 class Venv:
     """Creates and wraps the Python Virtual Environment to allow for code execution"""
 
+    PIP_TOOL: ClassVar[str] = os.environ.get(known_env.DLT_PIP_TOOL, None)
```

Comment:
Maybe DEPENDENCY_MANAGER? This communicates the purpose of the var more explicitly

Reply:
those are good points. but PIP means "Preferred Installer Program" so I think the name here... we can still change it before next release... maybe by expanding the acronym